terraform/paas/provider.tf

```diff
@@ -14,7 +14,6 @@ locals {
 }
 
 provider "azurerm" {
-  version                    = ">= 2.0"
```

Comment:
Why remove this line? Seems like a good practice to pin the provider version to a minimum version

Reply:
Deprecated - showing warnings